plugins/jobtype/src/azkaban/jobtype/HadoopHiveJob.java

```diff
@@ -22,12 +22,15 @@
 import java.io.IOException;
```

Comment:
make sense. We can open a github bug to track it and handle reportal changes separately.